README.md

```diff
@@ -103,6 +103,12 @@ user    0m0.018s
 sys     0m0.000s
 ```
 
+Since the CPython distribution was already downloaded and extracted (to
+`~/.nce/6c8db44ae0e18e320320bbaaafd2d69cde8bfea171ae2d651b7993d1396260b7` in this case) it is not
```

Comment:
I've explained a bit more back in scie-jump here: https://github.com/a-scie/jump/pull/42